### DISCUSSION TITLE
I made a toggle script for turning off/on vpn-kit from linux

### DISCUSSION THREAD

**Discussion by arizon-dread** [sakai135/wsl-vpnkit#179] [Show and tell]
```
cat /usr/local/bin/wsl-vpnkit-toggle
#!/bin/bash
if [ -z $1 ]
then
        echo "usage $0 start|stop"
        exit 1
fi

if [ "$1" = "start" ]
then
        $(sed -i 's/\#wsl\.exe/wsl\.exe/g' ~/.bashrc)
        $(wsl.exe -d wsl-vpnkit --cd /app service wsl-vpnkit start)
elif [ "$1" = "stop" ]
then
        $(sed -i 's/wsl\.exe/\#wsl\.exe/g' ~/.bashrc)
        $(wsl.exe -d wsl-vpnkit --cd /app service wsl-vpnkit stop)
fi
exit 0
```

this comments out the command/removes the comment "#" character on the wsl.exe lines in the files (/g for all occurances)

This is based on the assumption that this wsl.exe command is the only wsl.exe command in my .bashrc. If you're doing more magic with wsl.exe in your .bashrc, you need to ajust the sed command to be more specific.

**Comment by sakai135** [comment]
Nice, thanks for sharing.
